### PR TITLE
Package flac.0.5.0

### PR DESCRIPTION
**Pull request by toots**
### `flac.0.5.0`
Bindings to libflac

---
* Homepage: https://github.com/savonet/ocaml-flac
* Source repo: git+https://github.com/savonet/ocaml-flac.git
* Bug tracker: https://github.com/savonet/ocaml-flac/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

**Review by tmcgilchrist — COMMENTED** [review]
Also need  https://github.com/ocaml/opam-repository/pull/24360 to support building on Oracle Linux.

**Comment by toots** [comment]
Thanks for the review! Suggestion committed and pushed upstream as well.

**Comment by vitoyucepi** [comment]
Hi @toots,

Are you certain that the `GPL-2.0-only` license is correct? All the headers in the files indicate that they are licensed under `GPL-2.0-or-later`.

https://github.com/savonet/ocaml-flac/blob/7ac92bbb3c8441682bec5418d8c3457a7c61e2a7/src/flac_stubs.c#L1-L4





As far as I understand, the only difference between the licenses lies in the file header.

```diff

-This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; version 2.
+This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 2 of the License, or (at your option) any later version.

```

**Comment by mseri** [comment]
@vitoyucepi you are correct: https://github.com/savonet/ocaml-flac/blob/7ac92bbb3c8441682bec5418d8c3457a7c61e2a7/COPYING#L298

File: packages/flac/flac.0.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings to libflac"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0-only"
```
Comment (mseri) [inline packages/flac/flac.0.5.0/opam:6]:
```suggestion
license: "GPL-2.0-or-later"
```

**Comment by vitoyucepi** [comment]
Hi @mseri,
I think the changes have been made too quickly and may cause legal issues.
Before merging into the main branch, I would like to get approval from @toots as the lead developer.
Maybe I'm overcomplicating things, and everything will be fine regardless of the license.

**Comment by mseri** [comment]
I was going to wait for @toots anyway, I agree, but all files in the repository use this other license so the field in the opam file should be corrected

**Comment by mseri** [comment]
I modified the label so that it is clear that we are waiting for a confirmation

**Comment by toots** [comment]
Sorry y'all, I'm in and out of pocket right now. @vitoyucepi is spot on, as usual. I have fixed the upstream files. Thanks y'all.

**Comment by mseri** [comment]
Thanks!

**Pull request merged**